docs/roman.rst

```diff
@@ -12,35 +15,59 @@ Wide Field Instrument (WFI)
+The WFI model is based on the
+`Cycle 10 instrument reference information <https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html>`_
+from the Roman team at Goddard Space Flight Center (GSFC).
+The reported jitter for the Roman observatory is 0.012 arcsec per axis, `per GSFC <https://roman.ipac.caltech.edu/page/param-db#telescope>`_.
 
 .. note::
 
-   The current Roman WFI optical model was provided by Goddard Space Flight Center circa 2021 (the Cycle 9 reference data); a new optical model is currently being implemented in STPSF.
+   The current (Cycle 10) Roman WFI optical model was calculated by Goddard
+   Space Flight Center in September 2024.
 
 
-To work with the WFI model, import and instantiate it just like any of the JWST instruments::
+To work with the WFI model, import and instantiate it like any of the JWST instruments::
 
     >>> from stpsf import roman
     >>> wfi = roman.WFI()
 
-Usage of the WFI model class is, for the most part, just like any other STPSF instrument model. For help setting attributes like filters, position offsets, and sampling, refer to `Using STPSF <usage.html>`_.
+Usage of the WFI model class is, for the most part, just like any other STPSF
+instrument model. For help setting attributes like filters, position offsets,
+and sampling, refer to `Using STPSF <usage.html>`_.
 
-The WFI model includes a model for field dependent PSF aberrations. With as large a field of view as the WFI is designed to cover, there will be variation in the PSF from one end of the field of view to the other. STPSF's WFI model faithfully reproduces the field dependent aberrations calculated from the Goddard Roman team's Cycle 9 WFI design. This provides a toolkit for users to assess the impact of inter-SCA and intra-SCA PSF variations on science cases of interest.
+The WFI model includes a model for field dependent PSF aberrations. With as
+large a field of view as the WFI is designed to cover, there will be variation
+in the PSF from one end of the field of view to the other. STPSF's WFI model
+faithfully reproduces the field dependent aberrations calculated from the
+Goddard Roman team's Cycle 9 WFI design. This provides a toolkit for users to
+assess the impact of inter-detector and intra-detector PSF variations on
+science cases of interest.
 
 .. note::
 
    *Tutorial notebook for Roman*
 
-   This documentation is complemented by an `IPython Notebook tutorial for Roman PSFs <http://nbviewer.ipython.org/github/spacetelescope/stpsf/blob/stable/notebooks/STPSF-Roman_Tutorial.ipynb>`_. Download and run that notebook to use a beta notebook GUI for the WFI model, and to explore code samples for common tasks interactively.
+   This documentation is complemented by an
+   `IPython Notebook tutorial for Roman PSFs <http://nbviewer.ipython.org/github/spacetelescope/stpsf/blob/stable/notebooks/STPSF-Roman_Tutorial.ipynb>`_.
```

Comment:
This link leads to a 404 page. Is STPSF still working on NBViewer? If not, should we just link to the notebook's GitHub page? Or somewhere on the Roman Research Nexus?

Reply:
I think we should move STPSF-Roman_Tutorial.ipynb to the documentation and link to the readthedoc page.

Reply:
So it's recorded here, our preference is not to duplicate the notebook onto another webpage after [already having done so in RDox](https://roman-docs.stsci.edu/simulation-tools-handbook-home/stpsf-for-roman/stpsf-tutorials). I'm still not sure about the NBViewer link, but it seems like linking to the notebook's GitHub page might be best for now.

Reply:
I think the NBviewer link may just be an old remnant from back years ago before Github natively supported rendering notebooks